Formula/kamel.rb

```diff
@@ -41,7 +34,7 @@ def install
     assert_match "Error: cannot get command client: could not locate a kubeconfig", get_output
 
     version_output = shell_output("echo $(#{bin}/kamel version 2>&1)")
-    assert_match "1.0.0", version_output
+    assert_match "1.0.1", version_output
```

Comment:
```suggestion
    assert_match version.to_s, version_output
```

Reply:
thanks for the suggestion, I have made use of `version.to_s` now